### ISSUE TITLE
Suggestion - Give Huragok Engineers RCD

### ISSUE THREAD

**Issue by SharkLordSatan** [HaloSpaceStation/HaloSpaceStation13#3284]
As it currently stands, the Engineer module for Huragok already has a nice, wide array of tools at their disposal (although a large portion of these tools aren't actually used, such as stuff pertaining to wires/pipes/lights). They're a valuable asset when it comes to repairing vehicles and building barricades or walls.

However, when it comes to using them to, say, repair damaged walls on the Covenant ship, this is a **_painfully tedious and slow job_** that I would compare to being pure CBT unless you like zoning out for the next ten minutes as you place a fuckload of girders, then turn them into walls, and repeat, only for your work to be undone like five minutes later. This is because Huragok build objects _at the exact same speed_ as everyone else; thus, the only advantage they really have over a Yanme'e dragging around a crate full of construction materials is that they have access to basically unlimited nanolaminate and steel, and can survive in space without any EVA gear.

So how do you rectify this and make them _much_ more valuable, and thus an equally important role to a Huragok with the Lifeworker module?

Basically, my proposal is simple; we could give a custom RCD tool to the Engineer module, to allow Huragok to lay down walls _much, much more quickly_ and therefore making them a valuable asset at both plugging holes in the ship and setting up fortifications in the field. Having them be able to construct stuff such as barricades faster would also be nice, but I'm not sure if this is feasible without altering the build speed times for all people and not just Huragok.

**Comment by Aroliacue** [comment]
Done
